history/index.html

```diff
@@ -0,0 +1,661 @@
+Mac OS X disk image, a Linux/Unix self-extracting install, a new Quick
+Start welcome UI, Tip of the Day, support for many new key and
+certificate formats (PKI Path and SPC certs, Netscape SPKAC CSRs,
+OpenSSL keys), support to edit X.509 extensions and many more
+improvements too numerous to mention here. I also continued to improve
+the look and feel of the application with custom icons commissioned from
+<a href="http://www.iconaholic.com/">Iconaholic</a> (see the modern KSE
+application and KeyStore icons for examples of their awesome work) and
+continued by obsession with tweaking the splash screen.
+</p>
+<p>
+<img src="images/kse-xp-261.png" title="KSE 2.6.1 UI"
+class="img-fluid float-left mr-3" alt="KSE 2.6.1 UI Image" /> For the
+first time KSE felt to me like a grown-up desktop application. In
+addition I was miles ahead of any of the competition in terms of
+features (many open source keytool UIs had sprung up in the intervening
+years). I had invested heavily in terms of my time and reinvested some
+of the profits wisely in terms of icons, third party libraries and the
+many books I had studied over the years on general security, PKI, UI
+design and business. However, it was all worth it. Sales were still good
+and I could now say that I knew how to run a profitable business.
+</p>
+<p>
+Unfortunately by late 2009 a lot of the fun and excitement had
+evaporated. I had greatly enjoyed the experience up until then but was
+growing a little bored with running Lazgo Software. I had streamlined
+all I could but increasingly dreaded my quarterly weekend session with
+the accounts, dealing with the tax man and the plethora of other really
+dull things that are involved in running a business. Running the
+business was not a challenge any more, just a bit boring. Worse still I
+had no idea what direction I could take KSE in. It supported every
+relevant crypto format and was feature rich. It just seemed done to
+me.
+</p>
+<p>
+So the business definitely had to go. When something isn’t fun any
+more it isn’t worth it no matter the money it brings in (when I told
+people this at the time they looked at me like I was nuts, maybe I am).
+I still had to decide what to do with KSE, however. I will detail what
+happened next when I conclude this history in part three.
+</p>
+
+<h2>A History of KeyStore Explorer – Part Three (2012-05-18)</h2>
+<p>
+KeyStore Explorer (KSE) has existed, in one form or another, since
+2002. These days it is a freeware offering but it has not always been
+that way. KSE started as an open source project before morphing into a
+commercial project. It is only relatively recently that it was
+re-licensed to be free for all to use once again. As the utility is now
+almost ten years old I feel it is a good time to write a potted history
+of KSE.
+</p>
+<p>
+This post continues from <a
+href="#a-history-of-keystore-explorer-part-two-2012-02-21">Part Two</a>

```

Comment:
The link does not work, it should be `#a-history-of-keystore-explorer-–-part-two-2012-02-21`

There is another link to part 2 with the same problem. Most of the other links seem to work, but I haven't checked all of them.